ajax/libs/event-drops/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "event-drops",
+  "filename": "index.min.js",
```

Comment:
Please change to `"eventDrops.js"` instead.

Reply:
Hi @maruilian11 thanks for the review. Should it be eventDrops.min.js or eventDrops.js ?